package.json

```diff
@@ -7,6 +7,7 @@
     "url": "https://github.com/mozilla/blurts-server/issues"
   },
   "dependencies": {
```

Comment:
Can these move into `devDependencies`?

Reply:
Since we aren't shipping a compiled CSS file anymore, these are all needed for a production build.